PipelinesTasks/PRMetrics/buildAndReleaseTask/tests/invokers/azureReposInvoker.spec.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { WebApi } from 'azure-devops-node-api';
+import { IGitApi } from 'azure-devops-node-api/GitApi';
+import { IRequestHandler } from 'azure-devops-node-api/interfaces/common/VsoBaseInterfaces';
+import { CommentThreadStatus, GitPullRequest, GitPullRequestCommentThread, GitPullRequestIteration } from 'azure-devops-node-api/interfaces/GitInterfaces';
+import { expect } from 'chai';
+import { anyNumber, anyString, anything, instance, mock, verify, when } from 'ts-mockito';
+import AzureReposInvoker from '../../invokers/azureReposInvoker';
+import DevOpsApiWrapper from '../../wrappers/devOpsApiWrapper';
+import TaskLibWrapper from '../../wrappers/taskLibWrapper';
+import { resolvableInstance } from '../utils/resolvableInstance';
+
+const mockId = 3333;
+let azureReposInvoker: AzureReposInvoker;
+let taskLibWrapper: TaskLibWrapper;
+let devOpsApiWrapper: DevOpsApiWrapper;
+
+describe('azureReposInvoker.ts', function (): void {
```

Comment:
You can use `(): void =>` here

Reply:
I had a this.timeout in there and in an arrow function "this" can be null so the compiler gives you a warning about that. I removed the this.timeout so I can convert it now.